ApplicationConfig/Provider.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\PlatformUIBundle\ApplicationConfig;
+
+/**
+ * Provides parameters as a serializable value.
+ */
+interface Provider
+{
+    /**
+     * @return array|string|int|\JsonSerializable
```

Comment:
Nah, array won't do (anonymousUserId is a simple string, for instance). But it could be `mixed anything that can be serialized with json_encode()`